cpu/native/async_read.c

```diff
@@ -20,12 +20,16 @@
 #include <err.h>
 #include <signal.h>
 #include <stdlib.h>
+#include <string.h>
 #include <unistd.h>
 #include <fcntl.h>
 
 #include "async_read.h"
 #include "native_internal.h"
 
+#define ENABLE_DEBUG (0)
+#include "debug.h"
```

Comment:
I don't see, why this is required to be included now (maybe an artefact left from earlier DEBUG inclusion? `DEBUG()` isn't used in this file and it compiles without it.